buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -286,6 +298,20 @@ class ClusterFormationTasks {
         return copyPlugins
     }
 
+    static Task configureInstallModuleTask(String name, Project project, Task setup, NodeInfo node, Project module) {
+        if (node.config.distribution != 'integ-test-zip') {
+            throw new GradleException("Module ${module.path} cannot be installed in cluster which is not using integ-test-zip")
```

Comment:
`throw new GradleException("It doesn't make sense to install modules in distributions other than integ-test-zip because they should already have all modules bundled!")`